airbyte-integrations/connectors/source-monday/unit_tests/test_extractor.py

```diff
@@ -34,3 +34,27 @@ def test_extract_records():
     assert len(records) == 1
     assert records[0]["pulse_id"] == 123
     assert records[0]["created_at_int"] == 1636738688
+
+
+def test_extract_records_incremental():
+    # Mock the response
+    response = MagicMock()
+    response_body = {
+        "data": {
+            "boards": [
+                {
+                    "id": 1
+                }
+            ]
+        }
+    }
+
+    response.json.return_value = response_body
+    extractor = MondayIncrementalItemsExtractor(parameters={}, field_path=["data", "ccccc"], config=MagicMock())
+    extractor.field_path_pagination = ["data", "bbbb"]
+    extractor.field_path_incremental = ["data", "boards", "*"]
+    extractor.__post_init__(parameters={})
```

Comment:
Why do we call `__post_init__` manually?

Reply:
it is supposed that field_path_pagination and  field_path_incremental come from manifest.yaml.But here we have a deal with 'empty' object